docs/changelog.md

```diff
@@ -6,6 +6,9 @@ Change Log
 * Fix: Fix extension function imports (#1814).
 * Fix: Omit implicit modifiers on FileSpec.scriptBuilder (#1813).
 * Fix: Fix trailing newline in PropertySpec (#1827).
+Change: kotlinx-metadata 0.9.0. This is a breaking change for users of the
```

Comment:
This will only be breaking if the users of `:kotlinpoet-metadata` somehow forced the use of `kotlinx-metadata` 0.8.0 though, right? Otherwise, I don't think it's a breaking change, we're simply using a new API, but that's an implementation detail.

Reply:
Yes, it is only breaking if users of `:kotlinpoet-metadata` has other use of `kotlinx-metadata` where the `read` method is used (there might be other methods with this problem as well).

I put it in as `changelog.md` has: _Change: kotlinx-metadata 0.7.0. This is a breaking change for users of the `:kotlinpoet-metadata` module, as most `Flags`-API extensions have been removed in favor of the now-available first-party versions._ from when `kotlinx-metadata` was updated to 0.7.0.

Reply:
If I understand it correctly, `read` is deprecated, not removed entirely, right? If this is the case, it's not a breaking change, so I wouldn't mention it in the changelog.

With 0.7.0, it seems that we actually removed some `:kotlinpoet-metadata` API since they were replaced with similar first-party API, which is definitely a breaking change.